SConstruct

```diff
@@ -28,10 +28,15 @@ def add_sources(sources, directory):
             sources.append(directory + '/' + file)
 
 
+if platform == "osx":
```

Comment:
yeah it didn't make sense to me to keep repeating the same settings over and over again. What is the same regardless of what you're building should be outside of the target checks imho :) Within the target checks you add switches that only apply to that target. 

But thats my humble opinion :)